SecurityPkg/DeviceSecurity/SpdmSecurityLib/SpdmAuthentication.c

```diff
@@ -520,20 +519,25 @@ DoDeviceAuthentication (
     return EFI_DEVICE_ERROR;
   }
 
-  IsValidCertChain        = FALSE;
-  IsValidChallengeAuthSig = FALSE;
-  RootCertMatch           = FALSE;
+  *IsValidCertChain        = FALSE;
+  *RootCertMatch           = FALSE;
+  CertChainSize            = sizeof (CertChain);
+  ZeroMem (CertChain, sizeof (CertChain));
+  TrustAnchor     = NULL;
+  TrustAnchorSize = 0;
 
-  if (((CapabilityFlags & SPDM_GET_CAPABILITIES_RESPONSE_FLAGS_CERT_CAP) == 0) ||
-      ((CapabilityFlags & SPDM_GET_CAPABILITIES_RESPONSE_FLAGS_MEAS_CAP_SIG) == 0))

```

Comment:
why deleted this check `(CapabilityFlags & SPDM_GET_CAPABILITIES_RESPONSE_FLAGS_MEAS_CAP_SIG) == 0)` ?
If the device does not support MEAS_CAP with signature, AuthState should also being AUTH_FAIL_NO_SIG.

Reply:
Thanks. I have moved the check to the `DoDeviceMeasurement`.